doc/user_guide/big_data.rst

```diff
@@ -376,10 +376,10 @@ method can be used:
 
 .. versionadded:: 1.3.2
 
-By default, HyperSpy tries to optimize the chunking for most operations. However,
-it is sometimes possible to manually set a more optimal chunking manually. Therefore,
-many operations take a ``rechunk`` or ``optimize`` keyword argument to disable
-automatic rechunking.
+By default, HyperSpy does not change the chunking, as this can lead to reduced
+performance for large datasets. However, many operations take a ``rechunk`` or
+``optimize`` keyword argument which can be set to ``True`` to let HyperSpy
+change the chunking.
```

Comment:
I think that this is confusing because the version added is `.. versionadded:: 1.3.2`

Maybe add in a new block 

```suggestion
.. versionadded:: 2.0.0

Starting in version 2.0.0 HyperSpy does not automatically rechunk datasets as this leads for reduced performance.  The ``rechunk`` or ``optimize`` keyword argument can be set to ``True`` to change the chunking which could potentially speed up operations.
```

Reply:
Would it make sense to keep both?